src/main/resources/assets/design-system/icons.scss

```diff
@@ -96,3 +96,7 @@
 .sci-icon-product-cart-blue {
   background-image: url(svgResource('/assets/images/icons/cart.svg', #3465CC));
 }
+
+.sci-icon-product-cart-gray {
```

Comment:
Would use the same naming as the sirius-colors or otherwise it gets confusing as sirius-gray = #ACACAC. So maybe call this -muted

Reply:
The color comes from `sci-text-soft`.
The class `sci-muted-link` does not modify the color:
```
.sci-muted-link {
  text-decoration: none;
  cursor: pointer;

  &:hover {
    text-decoration: underline;
  }
}
```

Reply:
Should I replaces `-gray` with `-soft` then?

Reply:
Sorry yes, thats what I meant. Or `-text-soft`